sources/packages/forms/src/form-definitions/sfaa2022-23.json

```diff
@@ -5447,6 +5447,26 @@
           "properties": {},
           "lockKey": true,
           "defaultValue": "365"
+        },
+        {
+          "input": true,
+          "tableView": true,
+          "key": "currentTaxYear",
+          "label": "currentTaxYear",
+          "protected": false,
+          "unique": false,
+          "persistent": true,
```

Comment:
Please set the `persistent` to `false` to prevent saving it to the DB.

Reply:
Until or unless we use the hidden field in the validation of any other input, no need to have it persistent.